tests/bud.bats

```diff
@@ -2157,3 +2157,17 @@ EOM
   run_buildah inspect --format '{{ index .Docker.Config.Labels "io.buildah.version"}}' $target
   expect_output "$buildah_version"
 }
+
+@test "run check --from with arg" {
+        skip_if_no_runtime
+
+        ${OCI} --version
+        _prefetch alpine 
+        _prefetch debian
+
+        run_buildah bud --build-arg base=alpine --build-arg toolchainname=busybox --build-arg destinationpath=/tmp --pull=false --signature-policy ${TESTSDIR}/policy.json -f ${TESTSDIR}/bud/from-with-arg/Containerfile .
+	[ "$status" -eq 0 ]
```

Comment:
Fix spacing.

Reply:
Done, IDK how I managed to mess that up so badly without seeing it.